From 2e1ed56acd4d70b2d7ec99a6605068891d1857ef Mon Sep 17 00:00:00 2001
From: Ralf Habacker <ralf.habacker@freenet.de>
Date: Sat, 27 Apr 2019 19:28:28 +0200
Subject: Limit profiles selectable in the online quotes editor to the recent
 KDE major version

Due to a restriction within the knewstuff library it is currently
not possible to access the installation status of GHNS quotes of
the other KDE version.
---
 tools/onlinequoteseditor/mainwindow.cpp | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/tools/onlinequoteseditor/mainwindow.cpp b/tools/onlinequoteseditor/mainwindow.cpp
index 7571531..bb5bc0f 100644
--- a/tools/onlinequoteseditor/mainwindow.cpp
+++ b/tools/onlinequoteseditor/mainwindow.cpp
@@ -75,12 +75,15 @@ MainWindow::MainWindow(QWidget *parent)
     manager.setWebPageEnabled(true);
 
     manager.addProfile(new AlkOnlineQuotesProfile("no-config-file", AlkOnlineQuotesProfile::Type::None));
+#if QT_VERSION < QT_VERSION_CHECK(5,0,0)
     manager.addProfile(new AlkOnlineQuotesProfile("alkimia4", AlkOnlineQuotesProfile::Type::Alkimia4, "alkimia-quotes.knsrc"));
-    manager.addProfile(new AlkOnlineQuotesProfile("alkimia5", AlkOnlineQuotesProfile::Type::Alkimia5, "alkimia-quotes.knsrc"));
     manager.addProfile(new AlkOnlineQuotesProfile("skrooge4", AlkOnlineQuotesProfile::Type::Skrooge4, "skrooge-quotes.knsrc"));
-    manager.addProfile(new AlkOnlineQuotesProfile("skrooge5", AlkOnlineQuotesProfile::Type::Skrooge5, "skrooge-quotes.knsrc"));
     manager.addProfile(new AlkOnlineQuotesProfile("kmymoney4", AlkOnlineQuotesProfile::Type::KMyMoney4, "kmymoney-quotes.knsrc"));
+#else
+    manager.addProfile(new AlkOnlineQuotesProfile("alkimia5", AlkOnlineQuotesProfile::Type::Alkimia5, "alkimia-quotes.knsrc"));
+    manager.addProfile(new AlkOnlineQuotesProfile("skrooge5", AlkOnlineQuotesProfile::Type::Skrooge5, "skrooge-quotes.knsrc"));
     manager.addProfile(new AlkOnlineQuotesProfile("kmymoney5", AlkOnlineQuotesProfile::Type::KMyMoney5, "kmymoney-quotes.knsrc"));
+#endif
     d->ui.setupUi(this);
 
     d->quotesWidget = new AlkOnlineQuotesWidget(true, true);
-- 
cgit v1.1
